src/app/forms/templates/psv/psv-dda.template.ts

```diff
@@ -13,73 +14,139 @@ export const PsvDdaTemplate: FormNode = {
           name: 'certificateIssued',
           label: 'DDA certificate issued',
           value: '',
-          type: FormNodeTypes.CONTROL
+          type: FormNodeTypes.CONTROL,
+          viewType: FormNodeViewTypes.STRING,
```

Comment:
There is not need to specify `viewType: FormNodeViewTypes.STRING` since it's the default option